src/app/index.ts

```diff
@@ -274,7 +275,13 @@ module.exports = yo.extend({
           git().clone(projectRepoBranchInfo.repo, this.destinationPath(), ['--branch', projectRepoBranchInfo.branch || 'master'], async (err) => {
             // for all project types other than Excel Custom Functions. modify the generated project so it targets the selected host
             if (!this.project.isExcelFunctionsProject) {
-              await helperMethods.modifyProjectForSingleHost(this.destinationPath(), _.toLower(this.project.projectType), _.toLower(this.project.hostInternalName), language == 'ts');
+              // Call 'convert-to-single-host' npm script in generated project, passing in host parameter
+              const cmdLine = `npm run  --prefix ${this.destinationPath()} convert-to-single-host -- ${_.toLower(this.project.hostInternalName)}`;
```

Comment:
Did you override my last commit? I removed `--prefix` and added `--if-present` so there is no error if there is no script (since Excel-Custom-Functions doesn't need it).

Reply:
@TCourtneyOwen Please respond to this. You need to remove --prefix and add --if-present so it succeeds if no conversion is neccessary.

Reply:
this is updated - not sure why it's not showing here?